lib/utilities/pristine.js

```diff
@@ -246,6 +247,23 @@ function addEmberDataToDependencies(appName, version) {
   };
 }
 
+function removeEmberFromDependencies(appName, version) {
+  return function() {
+    var packageJSONPath = path.join(temp.pristinePath, appName, 'package.json');
+
+    var packageJSON = fs.readJsonSync(packageJSONPath);
+
+    if (version === 'canary' && packageJSON.devDependencies.hasOwnProperty('ember-source')) {
+      // ember-source does not support canary builds, therefore we will remove this entry and
+      // use ember from bower
+      debug('removing ember-source from NPM ');
+
+      delete packageJSON.devDependencies['ember-source'];
+      fs.writeJsonSync('package.json', packageJSON);
+    }
```

Comment:
It's already being added at a later point in the same function where this function is invoked.